jacoco-maven-plugin/src/org/jacoco/maven/DumpMojo.java

```diff
@@ -33,66 +36,81 @@
  * resource.
  * </p>
  * 
- * @goal dump
- * @phase post-integration-test
- * @threadSafe
  * @since 0.6.4
  */
+@Mojo(
+		name = "dump",
+		defaultPhase = LifecyclePhase.POST_INTEGRATION_TEST,
+		requiresProject = false,
```

Comment:
@marchof not intentional, my mistake actually - I thought that if javadoc `@requiresReports` is missing, then it is `false` by default, but in fact it is opposite. I'll fix this.